src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutes.scala

```diff
@@ -25,51 +25,55 @@ trait ManagedGroupRoutes extends UserInfoDirectives with SecurityDirectives with
   val managedGroupService: ManagedGroupService
 
   def groupRoutes: server.Route = requireUserInfo { userInfo =>
-    pathPrefix("group" / Segment) { groupId =>
-      val managedGroup = Resource(ManagedGroupService.managedGroupTypeName, ResourceId(groupId))
-
-      pathEndOrSingleSlash {
-        get {
-          handleGetGroup(managedGroup)
-        } ~
-        post {
-          handleCreateGroup(managedGroup, userInfo)
-        } ~
-        delete {
-          handleDeleteGroup(managedGroup, userInfo)
-        }
-      } ~
-      pathPrefix("requestAccess") {
-        post {
-          handleRequestAccess(managedGroup, userInfo)
-        }
-      } ~
-      pathPrefix(Segment) { policyName =>
-        val accessPolicyName = ManagedGroupService.getPolicyName(policyName)
+    (pathPrefix("groups" / "v1") | pathPrefix("group")) {
```

Comment:
Are the singular, non-versioned URIs deprecated?  If so, do we know how we want to go about marking them as such in the code, swagger, response payloads, etc.?